### ISSUE TITLE
Opcode list glitches out / gives exception and empty fields

### ISSUE THREAD

**Issue by ZainlessBrombie** [Col-E/Recaf#51]
When showing the opcode list of a method, I oftentimes get a number of elements that are missing all text content, including the number in front. The console shows the following warning:

```

(java:22546): Gtk-WARNING **: Unable to locate theme engine in module_path: "adwaita",
[INFO:5/28/18 11:37 PM] Loaded input from: 'BOSEconomy_0.7.8.1.jar'
Exception in thread "JavaFX Application Thread" java.lang.IllegalArgumentException
        at org.objectweb.asm.Type.getType(Type.java:443)
        at org.objectweb.asm.Type.getReturnType(Type.java:353)
        at org.objectweb.asm.Type.getReturnType(Type.java:555)
        at me.coley.recaf.ui.FormatFactory.addMethodType(FormatFactory.java:149)
        at me.coley.recaf.ui.FormatFactory.addOpcode(FormatFactory.java:203)
        at me.coley.recaf.ui.FormatFactory.opcode(FormatFactory.java:118)
        at me.coley.recaf.ui.component.InsnListEditor$OpcodeList.createFormat(InsnListEditor.java:541)
        at me.coley.recaf.ui.component.InsnListEditor$OpcodeList.lambda$10(InsnListEditor.java:514)
        at com.sun.javafx.application.PlatformImpl.lambda$null$117(PlatformImpl.java:295)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$118(PlatformImpl.java:294)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$null$450(GtkApplication.java:139)
        at java.lang.Thread.run(Thread.java:748)
[ERRR:5/28/18 11:37 PM] Could not locate: org.objectweb.asm.tree.VarInsnNode@211c7108 @1095
[ERRR:5/28/18 11:37 PM] Could not locate: org.objectweb.asm.tree.VarInsnNode@436ee718 @1100
```
java -version
java version "1.8.0_172"
Java(TM) SE Runtime Environment (build 1.8.0_172-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.172-b11, mixed mode)

also tried:

java -version
openjdk version "1.8.0_162"
OpenJDK Runtime Environment (build 1.8.0_162-8u162-b12-0ubuntu0.16.04.2-b12)
OpenJDK 64-Bit Server VM (build 25.162-b12, mixed mode)

Also I am on Linux Ubuntu 16 with Kubuntu-desktop installed on top of it.

**Comment by Col-E** [comment]
Its an issue with the format renderer. Looks like your sample is public, so I'll grab it and fix the issue.

**Comment by ZainlessBrombie** [comment]
Yes I can. The code is from "BOSEconomy", which has been discontinued (which is why I'm patching it for a friend who uses it), which fortunately is now in the public domain. 

uploaded it here:
http://www.zainlessbrombie.com/downloads/other/BOSEconomy.jar

Method in question:
cosine/boseconomy/bracket/AccountGroup/addAccounts

Illegal character (used the recaf editor to add a message to the exception because, well, reasons): 'j'
No clue why its trying to read a lowercase j.

Oh by the way you called the new release file Recaf.jar.jar :P

**Comment by Col-E** [comment]
Oh the code in this jar isn't valid bytecode.

The descriptor is missing `L`

What it is: `()java/util/Collection;`
What it should be: `()Ljava/util/Collection;`

Another error: `()Ljava/lang/Object`
Should be: `()[Ljava/lang/Object;`

**Comment by Col-E** [comment]
Resolved as of e6207cd1f03c13a47292d48432bf618c3988785a

**Comment by ZainlessBrombie** [comment]
Thats... interesting ô_o
Thanks <3
